chart/kubeapps/README.md

```diff
@@ -242,6 +242,29 @@ kubectl apply -f <repo name>.yaml
 
 After that you should be able to access the new version of Kubeapps. If the above doesn't work for you or you run into any other issues please open an [issue](https://github.com/kubeapps/kubeapps/issues/new).
 
+### Upgrading to 5.0.0
+
+[On November 13, 2020, Helm v2 support was formally finished](https://github.com/helm/charts#status-of-the-project), this major version is the result of the required changes applied to the Helm Chart to be able to incorporate the different features added in Helm v3 and to be consistent with the Helm project itself regarding the Helm v2 EOL.
```

Comment:
Hmm, I'm guessing this comes from a standard text you're adding to lots of charts? I'd personally reword the first statement to "On November 13, 2020, Helm 2 support ended" . The important part is just "Helm 2" rather than "Helm v2", since the "v2" really refers to the chart API version which is actually the most recent one.

Reply:
I just replaced all the Helm vX with Helm X, it was copied from the official doc, they use both nomenclatures to refer to the Helm tool itself, for example, https://helm.sh/blog/covid-19-extending-helm-v2-bug-fixes/
> When Helm v3 was released in November 2019...

In this case, it can't be referred to apiVersion since v3 doesn't exist. 

But yes, to avoid confusion I just removed the `v`